frodo.js

```diff
@@ -5,89 +5,90 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-Consider renaming this variable to something more descriptive for better readability.